### PR TITLE
Fix test name typos

### PR DESCRIPTION
**Pull request by tjoyal**
Terrible typo on my part. 

:eyes: @sgnr @davidcornu

### CODE REVIEW

**Comment by sgnr** [comment]
Shame on us reviewers for not having caught that :shipit:

File: spec/money_accessor_spec.rb [hunk 1]
```diff
@@ -34,35 +34,35 @@ def initialize(price)
     expect(object.price).to eq(nil)
   end
 
-  it "generates an attribute weitter that allow setting a money object" do
+  it "generates an attribute writter that allow setting a money object" do
```
Comment (davidcornu) [inline spec/money_accessor_spec.rb]: I think it's "writer"

Comment (sgnr) [inline spec/money_accessor_spec.rb]: Spelling is hard.

**Comment by davidcornu** [comment]
Fix the new typo that replaced the old typo :trollface: and :shipit:

File: spec/money_accessor_spec.rb (hunk 1 above)
Comment (tjoyal) [inline spec/money_accessor_spec.rb]: Of course :sadpanda:

**Pull request merged**